content/en/blog/_posts/2019-06-05-what-and-the-why-of-cluster-api.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog
+title: 'The What and the Why of the Cluster API'
+date: 2019-06-05
+---
+
+**Author**: Tim St. Clair, Senior Staff Engineer at VMware
+
+_Note: This post was originally published on the [VMware Cloud Native Apps blog](https://blogs.vmware.com/cloudnative/2019/03/14/what-and-why-of-cluster-api/) and republished here with author's permission_
```

Comment:
Because this post talks about things that are now in the past, does it make sense to clarify _when_ it was published?

Reply:
done